### PR TITLE
ROC-4965: Update paid in full staff email to include date

### PR DESCRIPTION
**Pull request by laurentaylor1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-4965

### Change description ###
Date money was received on added to staff email for paid in full.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class PaidInFullStaffNotificationServiceTest extends MockSpringTest {
+
+    private static final byte[] PDF_CONTENT = {1, 2, 3, 4};
+
+    @Captor
+    private ArgumentCaptor<String> senderArgument;
```
Comment (andrewwa-kainos) [inline src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java]: This is not needed. Just compare the value directly in the `verify()` calls (with `eq()` for example)

File: src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+        verify(emailService).sendEmail(senderArgument.capture(), emailDataArgument.capture());
+
+        assertThat(emailDataArgument.getValue()
+            .getSubject()).startsWith("Paid in Full");
+        assertThat(emailDataArgument.getValue()
+            .getMessage()).contains(
+            "01/12/2025");
+    }
+}
```
Comment (andrewwa-kainos) [inline src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java]: Any error scenarios to test?

File: src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+        service.notifyPaidInFull(claimWithPaidInFull);
+
+        verify(emailService).sendEmail(senderArgument.capture(), emailDataArgument.capture());
+
+        assertThat(emailDataArgument.getValue()
+            .getSubject()).startsWith("Paid in Full");
+        assertThat(emailDataArgument.getValue()
+            .getMessage()).contains(
+            "01/12/2025");
```
Comment (andrewwa-kainos) [inline src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java]: Sort out the formatting here please

File: src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java (hunk 2 above)
Comment (laurentaylor1) [reply src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java]: Nope check with Rob, null value of moneyReceivedOn should never reach this far

File: src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java (hunk 1 above)
Comment (laurentaylor1) [reply src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java]: Hey @andrewwa-kainos could you expand on this please? Thanks

Comment (andrewwa-kainos) [reply src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java]: You only need to capture an argument if you're then going to examine it. In this case you're capturing it on line 51 (at time of writing) but then not doing anything with it. You could remove this field and then either test its value (which would be new) with `eq(whateverItsMeantToBe)` or just leave it as `anyString()`

Comment (laurentaylor1) [reply src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java]: Ah right gotcha cheers

Comment (laurentaylor1) [reply src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java]: Done :) @andrewwa-kainos

**Pull request merged**